### ISSUE TITLE
Deprecation Notice: Class Crazymeeks\Foundation\Adapter\SoapClientAdapter

### ISSUE THREAD

**Issue by sironronron** [crazymeeks/dragonpay#11]
I'm getting these warnings whenever I will run composer:

```
Deprecation Notice: Class Crazymeeks\Foundation\Adapter\SoapClientAdapter located in D:/projects/taxheroes/vendor/crazymeeks/dragonpay/src\Adapter\SoapClientAdapter.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar://C:/ProgramData/ComposerSetup/bin/composer.phar/src/Composer/Autoload/ClassMapGenerator.php:201
Deprecation Notice: Class Crazymeeks\Foundation\PaymentGateway\Dragonpay\Action\ActionInterface located in D:/projects/taxheroes/vendor/crazymeeks/dragonpay/src\Foundation\PaymentGateway\DragonPay\Action\ActionInterface.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar://C:/ProgramData/ComposerSetup/bin/composer.phar/src/Composer/Autoload/ClassMapGenerator.php:201
```

**Comment by crazymeeks** [comment]
This composer warning will be fixed in the next patch!

Thanks for reporting!

**Comment by crazymeeks** [comment]
This warning had been fixed! Use patch v3.2.6!
